tests/DfE.FindInformationAcademiesTrusts.UnitTests/Pages/Trusts/Contacts/EditSfsoLeadModelTests.cs

```diff
@@ -78,7 +78,7 @@ public async Task OnPostAsync_sets_ContactUpdated_to_true_when_validation_is_cor
 
         result.Should().BeOfType<RedirectToPageResult>();
         var redirect = result as RedirectToPageResult;
-        redirect!.PageName.Should().Be("/Trusts/Contacts");
+        redirect!.PageName.Should().Be("/Trusts/Contacts/InDfe");
```

Comment:
Info (not requested change) - I recently discovered that these kinds of assertions following a type can be written more safely and succinctly:

```csharp
        var redirect = result.Should().BeOfType<RedirectToPageResult>().Subject;
        redirect.PageName.Should().Be("/Trusts/Contacts/InDfe");
```